app/src/main/java/com/iron/espresso/presentation/place/SearchPlaceDetailViewModel.kt

```diff
@@ -0,0 +1,52 @@
+package com.iron.espresso.presentation.place
+
+import android.util.Log
+import androidx.hilt.lifecycle.ViewModelInject
+import androidx.lifecycle.MutableLiveData
+import com.iron.espresso.base.BaseViewModel
+import com.iron.espresso.data.model.LocalItem
+import com.iron.espresso.model.api.KakaoApi
+import com.iron.espresso.model.response.LocalResponse
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+class SearchPlaceDetailViewModel @ViewModelInject constructor(private val kakaoApi: KakaoApi) :
```

Comment:
```suggestion
class SearchPlaceDetailViewModel @ViewModelInject constructor(private val placeRepository: PlaceRepository) :
```
이쪽도 모델 구성 해보면 좋을 것 같아요.

Reply:
당장은 아니더라도